scripts/ga4gh/tool_execution_challenge_phase1/empty.options.json

```diff
@@ -0,0 +1,3 @@
+{
+
+}
```

Comment:
Consider this optional to actually change but a good rule of thumb is to end files with a `\n` as if you cat them your prompt winds up on its own line, which is handy